gcp_flowlogs_reader/gcp_flowlogs_reader.py

```diff
@@ -99,6 +100,7 @@ def __init__(self, entry: StructEntry):
         )
 
         self.bytes_sent = int(flow_payload['bytes_sent'])
+        self.compressed_bytes_sent = int(flow_payload['compressed_bytes_sent'])
```

Comment:
So the `compressed_bytes_sent` isn't a property of the flow itself, but rather the transfer of flows from the GCP service to our service that we want to measure.

In the [AWS edition](https://github.com/obsrvbl/flowlogs-reader/pull/53/files#diff-94c558263d20eb69c3f3b487e7c187b06005dcac89dd257960381d639a024f5fR223) you can see that the `bytes_processed` is a property added to the `BaseReader` class, not the `FlowRecord` itself.

I'd expect the increment to happen in the vicinity of the `page_helper` call, which is somewhat analogous to the [AWS version](https://github.com/obsrvbl/flowlogs-reader/pull/53/files#diff-94c558263d20eb69c3f3b487e7c187b06005dcac89dd257960381d639a024f5fR333-R334).